src/funfuzz/js/compare_jit.py

```diff
@@ -109,7 +110,7 @@ def compareLevel(jsEngine, flags, infilename, logPrefix, options, showDetailedDi
         if (r.return_code == 1 or r.return_code == 2) and (anyLineContains(r.out, "[[script] scriptArgs*]") or (
                 anyLineContains(r.err, "[scriptfile] [scriptarg...]"))):
             print("Got usage error from:")
-            print("  %s" % sps.shellify(command))
+            print("  %s" % " ".join([quote(x) for x in command]))
```

Comment:
Don't use `str.join([`*`list comprehension`*`])`, use `str.join(`*`generator expression`*`)` instead.
List comprehension builds the whole list in-memory and then calls `join()` on the result. A generator expression works as though each item is yielded to `join()` as it is evaluated. The syntax is the same, just drop the square brackets.

This occurs on nearly every line changed, so I haven't added comments on them all.